pkg/internal/approver/allowed/validation.go

```diff
@@ -76,14 +76,28 @@ func (a allowed) Validate(_ context.Context, policy *policyapi.CertificateReques
 	}
 
 	for _, stringSlice := range stringSlices {
-		if stringSlice.slice != nil && stringSlice.slice.Required != nil && *stringSlice.slice.Required && stringSlice.slice.Values == nil {
-			el = append(el, field.Required(stringSlice.path.Child("values"), "values must be defined if required field"))
+		if stringSlice.slice != nil {
+			if stringSlice.slice.Required != nil && *stringSlice.slice.Required && stringSlice.slice.Values == nil {
+				el = append(el, field.Required(stringSlice.path.Child("values"), "values must be defined if required field"))
```

Comment:
i know you aren't changing this text, but I don't like this text :(

Reply:
I agree, but I am trying hard to not get an XXXXXXXXL PR. 😉

Reply:
I would prefer to not change unrelated stuff in this already huge PR. Hope it's ok.